libs/ml/include/ml/ops/ops.hpp

```diff
@@ -149,6 +151,60 @@ class Ops
     // Empty deafult implementation for non-trainable Ops.
   }
 
+  /**
+   * @brief ChargeForward
+   * @return estimated charge amount, necessary for performing a forward pass on data of given
+   * shapes.
+   */
+  virtual MLChargeAmount ChargeForward()
+  {
+    // TODO(VH): make a pure virtual call;
+    FETCH_LOG_ERROR(Descriptor(),
+                    " Error: call to unexisting ChargeForward() implementation! returned 0.");
+    return 0;
+  }
+
+  /**
+   * @brief ChargeBackward
+   * @return estimated charge amount, necessary for performing a backward pass on data of given
+   * shapes.
+   */
+  virtual MLChargeAmount ChargeBackward() const
+  {
+    // TODO(VH): make a pure virtual call;
```

Comment:
again... why not just do this?

Reply:
the same as above, creating dummy empty override in each Ops will only create a huge change without any profit. It is better to do it in scope of implementation of estimation for each particular op

Reply:
ML-483 created for this, in order to prevent forgetting :)